### PR TITLE
Add variants table

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

Add the variants table to the covid-variants page:

image

Mobile view:

image

## Motivation

Feature request

### Governance

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: packages/app/src/domain/variants/variants-table-tile/components/percentage-bar-with-number.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  return (
+    <Box
+      color={color}
+      display="flex"
+      alignItems="center"
+      pr={{ _: 0, sm: 2, lg: 4, xl: 5 }}
+    >
+      <InlineText fontWeight="bold" color="black" pr={2}>
+        {`${formatPercentage(percentage)}%`}
+      </InlineText>
```
Comment (VWSCoronaDashboard16) [inline packages/app/src/domain/variants/variants-table-tile/components/percentage-bar-with-number.tsx:23]: I think this should be with one decimal, a fixed width & aligned right to match designs.

File: packages/app/src/domain/variants/variants-table-tile/components/percentage-bar-with-number.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      <PercentageBar percentage={percentage} height="8px" />
```
Comment (VWSCoronaDashboard16) [inline packages/app/src/domain/variants/variants-table-tile/components/percentage-bar-with-number.tsx:24]: In the design the bar has a gray background

File: packages/app/src/domain/variants/variants-table-tile/components/variant-difference.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  const diffText = siteText.covid_varianten.varianten_tabel.verschil;
+
+  if (value === undefined) {
+    return <>-</>;
+  }
+  if (value.difference > 0) {
+    return (
+      <Difference color={colors.body}>
+        <PijlOmhoog />
+        {formatPercentage(value.difference)}% {diffText.meer}
```
Comment (VWSCoronaDashboard16) [inline packages/app/src/domain/variants/variants-table-tile/components/variant-difference.tsx:21]: This also has decimals in the design

File: packages/app/src/domain/variants/variants-table-tile/components/mobile-variants-table.tsx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+            <div ref={ref}>
+              <Box mb={1}>
+                {columnNames['aantal_monsters']}:{' '}
+                <NumberOfSamples
+                  occurrence={row.occurrence}
+                  sampleSize={row.sampleSize}
+                />
+              </Box>
+              <Box mb={1}>
+                {columnNames['vorige_meeting']}:{' '}
```
Comment (VWSCoronaDashboard16) [inline packages/app/src/domain/variants/variants-table-tile/components/mobile-variants-table.tsx:98]: I think this should be 'meting'

**Review by VWSCoronaDashboard16 — CHANGES_REQUESTED** [review]
Code 👌 
Just some points on aesthetics and copy

File: packages/app/src/domain/variants/variants-table-tile/components/percentage-bar-with-number.tsx (hunk 2 above)
Comment (VWSCoronaDashboard1) [reply packages/app/src/domain/variants/variants-table-tile/components/percentage-bar-with-number.tsx:24]: The design differs from the rest of the site, so I ignored that bit ;)

File: packages/app/src/domain/variants/variants-table-tile/components/percentage-bar-with-number.tsx (hunk 1 above)
Comment (VWSCoronaDashboard1) [reply packages/app/src/domain/variants/variants-table-tile/components/percentage-bar-with-number.tsx]: Fixed!

File: packages/app/src/domain/variants/variants-table-tile/components/variant-difference.tsx (hunk 3 above)
Comment (VWSCoronaDashboard1) [reply packages/app/src/domain/variants/variants-table-tile/components/variant-difference.tsx]: Fixed!

File: packages/app/src/domain/variants/variants-table-tile/components/mobile-variants-table.tsx (hunk 4 above)
Comment (VWSCoronaDashboard1) [reply packages/app/src/domain/variants/variants-table-tile/components/mobile-variants-table.tsx:98]: You're right, but I've already imported this as a Lokalize key so I'm leaving this in for now. It's not a public-facing typo, so I can live with it.

Comment (VWSCoronaDashboard16) [reply packages/app/src/domain/variants/variants-table-tile/components/mobile-variants-table.tsx:98]: I meant that the actual text value itself has the same typo

image

Comment (VWSCoronaDashboard1) [reply packages/app/src/domain/variants/variants-table-tile/components/mobile-variants-table.tsx:98]: Ah ok, I'll update that in Sanity. But still, all these values are changed by the communication team anyway before we release, so any mistakes we make there aren't a huge issue.

Comment (VWSCoronaDashboard16) [reply packages/app/src/domain/variants/variants-table-tile/components/mobile-variants-table.tsx:98]: Aah ok, good to know 👍

**Review by VWSCoronaDashboard16 — APPROVED** [review]

File: packages/app/src/domain/variants/variants-table-tile/logic/use-variants-table-data.ts [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+ * Finally the list gets sorted with the highest percentage on top and
+ * the 'other' variant always last in the list.
+ */
+export function useVariantsTableData(
+  data: NlVariantsValue,
+  countriesOfOrigin: CountriesOfOrigin,
+  differences: NationalDifference
+): VariantRow[] {
+  return useMemo(() => {
+    return variants
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/domain/variants/variants-table-tile/logic/use-variants-table-data.ts:51]: nit;
```suggestion
  return useMemo(() => variants
```

**Review by VWSCoronaDashboard8 — APPROVED** [review]

**Pull request merged**